applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -4,3 +4,399 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+#if (CONFIG_AUDIO_DEV == GATEWAY)
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/audio/audio.h>
+
+#include "le_audio.h"
+#include "macros_common.h"
+#include "ctrl_events.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(cis_gateway, CONFIG_LOG_BLE_LEVEL);
+
+static struct bt_conn *default_conn;
+static struct bt_audio_stream audio_stream;
+static struct bt_audio_unicast_group *unicast_group;
+static struct bt_codec *remote_codecs[CONFIG_BT_AUDIO_UNICAST_CLIENT_PAC_COUNT];
+static struct bt_audio_ep *sinks[CONFIG_BT_AUDIO_UNICAST_CLIENT_ASE_SNK_COUNT];
+
+#define BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO                                                  \
+	BT_AUDIO_LC3_PRESET(BT_CODEC_LC3_CONFIG_48_4,                                              \
+			    BT_CODEC_LC3_QOS_10_UNFRAMED(120u, 2u, 20u, 10000u))
+#define HCI_ISO_BUF_ALLOC_PER_CHAN 2
+#define NET_BUF_POOL_ITERATE(i, _)                                                                 \
+	NET_BUF_POOL_FIXED_DEFINE(iso_tx_pool_##i, HCI_ISO_BUF_ALLOC_PER_CHAN,                     \
+				  BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_TX_MTU), 8, NULL);
+#define NET_BUF_POOL_PTR_ITERATE(i, ...) IDENTITY(&iso_tx_pool_##i)
+
+LISTIFY(CONFIG_BT_ISO_MAX_CHAN, NET_BUF_POOL_ITERATE, (;))
+
+#define BT_LE_CONN_PARAM_MULTI                                                                     \
+	BT_LE_CONN_PARAM(CONFIG_BLE_ACL_CONN_INTERVAL, CONFIG_BLE_ACL_CONN_INTERVAL,               \
+			 CONFIG_BLE_ACL_SLAVE_LATENCY, CONFIG_BLE_ACL_SUP_TIMEOUT)
+
+#define DEVICE_NAME_PEER "NRF5340_AUDIO"
+#define DEVICE_NAME_PEER_LEN (sizeof(DEVICE_NAME_PEER) - 1)
+static struct net_buf_pool *iso_tx_pools[] = { LISTIFY(CONFIG_BT_ISO_MAX_CHAN,
+						       NET_BUF_POOL_PTR_ITERATE,
+						       (,)) };
+
+static struct bt_audio_lc3_preset unicast_present_nrf5340 =
+	BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO;
+
+static void start_scan(void);
+
+static int unicast_group_create(struct bt_audio_stream *stream)
+{
+	int ret;
+
+	ret = bt_audio_unicast_group_create(stream, CONFIG_BT_ISO_MAX_CHAN, &unicast_group);
+	if (ret != 0) {
```

Comment:
I suggest using if (ret) in stead of if(ret != 0).
This occurs several places in this file.